src/geo/spatial.clj

```diff
@@ -93,7 +100,16 @@
 
   com.vividsolutions.jts.geom.Geometry
   (to-shape [this]
-           (JtsGeometry. this jts-earth true)))
+    ;; Cloning geometries that cross dateline to workaround
+    ;; spatial4j / jts conversion issue: https://github.com/locationtech/spatial4j/issues/150
+    (let [geom (if (crosses-dateline? this)
+                 (.clone this)
+                 this)]
```

Comment:
A way to make code self-documenting is to define named variables for the literals you'd use. So here, you can add to your `let`:

```clojure
(let [dateline-check? true
      allow-multi-overlap? false]
  ...
)
```